### ISSUE TITLE
mainDashboard: Release Notes & Message from Envisioning (Closed Beta)

### ISSUE THREAD

**Issue by arthursoares** [envisioning/tdb#258]
Related to #257 

I think it would be a good addition to have a TDB recent updates box. With content related to release news and some system wide messages. 

I think for the closed beta it is nice to have a `hero ibox` with some information about everything.

For release news/updates, we could just have a notification on the right side with a link to a github post or pull request.

What do you think? @rafaelcorreiapoli @lnmunhoz @michellzappa

**Comment by rafaelcorreiapoli** [comment]
The content would be published manually via some admin's page, right?
How would that notificaiton on the right side work? would it flash out or be permanent? Could you give me some reference design ?

**Comment by arthursoares** [comment]
@rafaelcorreiapoli Yes.

I was looking at the resources from Inspinia. I think we could use the `sweet alert` with a "read more" and "Ok" button.

**Comment by rafaelcorreiapoli** [comment]
So whenever the user access tdb and there are some news, the sweet alert appears. When he clicks Ok or Read more, he won't see this popup anymore until next the update.
We must track when the user saw/clicked the Sweet alert or no to show this only once for each user

**Comment by michellzappa** [comment]
Yes! Very useful!
Could be structured information from some source.

**Comment by rafaelcorreiapoli** [comment]
@arthursoares Poderia me dar uma especificação mais detalhada dos requisitos e implementação desta funcionalidade?
